apps/web/src/server/applications/case/representations/application-representations.view-model.js

```diff
@@ -0,0 +1,83 @@
+import { format } from 'date-fns';
```

Comment:
some of these functions can be probably found in the nunjucks filters folder 'web/src/server/lib".'

For example I am sure the date formatting has been implemented already